src/sections/General/Navigation/navigation.style.js

```diff
@@ -11,6 +11,9 @@ const NavigationWrap = styled.header`
 
   .nav-container {
     display: flex;
+    @media (min-width:768px){
```

Comment:
It should be limited from `912px to 992px` as for other screen resolutions it is constrained by the `Container` component.